data/caasp/update.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+set -ex
+
+# 2.0 -> 2.0 http://download.suse.de/ibs/Devel:/CASP:/2.0:/ControllerNode:/TestUpdate/standard/Devel:CASP:2.0:ControllerNode:TestUpdate.repo
+# 
+
+while getopts "r:" opt; do
+case $opt in
+	r)
+		REPO=$OPTARG
+		;;
+	:)
+		echo "Option -$opt requires an argument." >&2
+		;;
+esac
+done
+
+saltid=$(docker ps | grep salt-master | awk '{print $1}')
+where="-P roles:(admin|kube-(master|minion))"
+srun="docker exec -i $saltid salt --batch 7"
+runner="$srun $where cmd.run"
+
+# Remove non-existent ISO repository
+# $runner "zypper rr 1"
+
+# Add repository as system venodors
+$runner 'echo -e "[main]\nvendors = suse,opensuse,obs://build.suse.de,obs://build.opensuse.org" > /etc/zypp/vendors.d/vendors.conf'
+$runner "zypper ar --refresh --no-gpgcheck $REPO"
+
+# Manually Trigger Transactional Update (or wait up to 24 hours for it run by itself)
+$runner 'systemctl disable --now transactional-update.timer'
+$runner '/usr/sbin/transactional-update cleanup dup salt'
+
+# Manually Refresh the Grains (or wait up to 10 minutes)
+$srun '*' saltutil.refresh_grains
+
+
+
+# Check after update (reboot)
+
+# # caasp-container-manifests
+# # check installed version
```

Comment:
I will add my tests for that later. Let's make baby tests first, to make sure that we can update so far. Could you please add a `TODO` in front of the comment, in order to highlight this as WIP?

Reply:
I uncommented, I want to have basic check as part of this PR